core/controllers/admin_test.py

```diff
@@ -1047,3 +1047,26 @@ def test_clear_search_index(self):
         self.assertEqual(result_explorations, [])
         result_collections = search_services.search_collections('Welcome', 2)[0]
         self.assertEqual(result_collections, [])
+
+
+class SendDummyMailTest(test_utils.GenericTestBase):
+    """"Tests for sending test mails to admin."""
+
+    def setUp(self):
+        super(SendDummyMailTest, self).setUp()
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+
+    def test_send_dummy_mail(self):
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+        csrf_token = self.get_new_csrf_token()
+        generated_response = self.post_json(
+            '/sendDummyMailToAdminHandler', payload={},
+            csrf_token=csrf_token)
+        if feconf.CAN_SEND_EMAILS:
```

Comment:
No if/else in tests. Have separate tests for the "can send" and "cannot send" case, and swap out feconf.CAN_SEND_EMAILS to the appropriate value for each test.

Reply:
Done.